cmd/admin/main.go

```diff
@@ -92,6 +93,7 @@ func main() {
 	// they're present.
 	configFile := flag.String("config", "", "Path to the configuration file for this service (required)")
 	dryRun := flag.Bool("dry-run", true, "Print actions instead of mutating the database")
+	debugAddr := flag.String("debug-addr", "", "Debug server address override")
```

Comment:
If we're removing the need for debugAddr in `newStatsRegistry`, then I think there's no need to add this CLI flag here anymore.

Reply:
I agree with this. And above, in admin.go lines 50-52.

Reply:
The newest commit deprecates and stops any use of debugAddr from the config file, and just uses the CLI flag. Is this OK, or are you saying it would make sense to stop plumbing through any debugAddr at all for `admin`?

Reply:
I'm saying it would make sense to stop plumbing through any debugAddr at all for `admin`.